pkg/rancher-desktop/assets/scripts/nginx.conf

```diff
@@ -0,0 +1,175 @@
+worker_processes  auto;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    map_hash_bucket_size 128;
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    log_format proxy escape=json
+      '{'
+        '"access_time":"$time_local",'
+        '"request":"$request",'
+        '"status":"$status",'
+        '"bytes_sent":"$body_bytes_sent",'
+        '"host":"$host",'
+        '"ssl_protocol":"$ssl_protocol",'
+        '"connect_host":"$connect_host",'
+        '"connect_port":"$connect_port",'
+      '}';
+
+    log_format mitm escape=json
+      '{'
+        '"access_time":"$time_local",'
+        '"method":"$request_method",'
+        '"uri":"$uri",'
+        '"status":"$status",'
+        '"bytes_sent":"$body_bytes_sent",'
+        '"upstream_response_time":"$upstream_response_time",'
+        '"host":"$host",'
+        '"http_host":"$http_host",'
+        '"upstream":"$upstream_addr"'
+      '}';
+
+    gzip off;
```

Comment:
Hmm, any particular reason gzip is off? Might be good to have a comment here describing why.

Reply:
It is just the default setting. I will delete the line.

Reply:
Done